src/Microsoft.Net.Http.Headers/ContentDispositionHeaderValue.cs

```diff
@@ -69,13 +69,14 @@ public IList<NameValueHeaderValue> Parameters
 
         public StringSegment Name
         {
-            get { return GetName(NameString); }
+            get { return HeaderUtilities.RemoveQuotes(GetName(NameString)); }
```

Comment:
You mean GetName should return a StringSegment. I guess RemoveQuotes could be moved into GetName.